src/main/java/net/fabricmc/installer/client/ClientHandler.java

```diff
@@ -52,13 +57,33 @@ public void install() {
 				if (createProfile.isSelected()) {
 					ProfileInstaller.setupProfile(mcPath, profileName, gameVersion);
 				}
+				SwingUtilities.invokeLater(() -> showInstalledMessage(loaderVersion, gameVersion));
 			} catch (Exception e) {
 				error(e);
 			}
 			buttonInstall.setEnabled(true);
 		}).start();
 	}
 
+	private void showInstalledMessage(String loaderVersion, String gameVersion) {
+		JEditorPane pane = new JEditorPane("text/html", new MessageFormat(Utils.BUNDLE.getString("prompt.install.successful")).format(new Object[]{loaderVersion, gameVersion, Reference.fabricApiUrl}));
+		pane.setEditable(false);
+		pane.addHyperlinkListener(e -> {
+			try {
```

Comment:
weird spacing on this line. could conflict with style of code

Reply:
This is just one tab indent. Look at the line above, this is a new code block so it needs to indent